caffe2/mkl/operators/local_response_normalization_op.cc

```diff
@@ -0,0 +1,86 @@
+#include "caffe2/core/context.h"
+#include "caffe2/core/operator.h"
+#include "caffe2/operators/local_response_normalization_op.h"
+
+#include "caffe2/utils/mkl_utils.h"
+
+#ifdef CAFFE2_HAS_MKL_DNN
+
+namespace caffe2 {
+namespace mkl {
+
+template <typename T>
+class MKLLRNOp final : public LRNOpBase<T, MKLContext>{
+
+  public:        
+    MKLLRNOp(const OperatorDef &operator_def, Workspace *ws)
+        : LRNOpBase<T, MKLContext>(operator_def, ws) {}
+
+    ~MKLLRNOp() {
+        if (workspace_buffer_ != NULL) {
```

Comment:
could you use clang-format  ? I have sent you an e-mail with more detailed instructions.